thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -168,7 +224,8 @@ static ServiceInfo newServiceInfo(
             });
         }
 
-        return new ServiceInfo(name, functions, classes, endpoints, docStrings.get(name), exampleHeaders);
+        return new ServiceInfo(name, functions, classes, endpoints,
```

Comment:
I'm thinking of adding the following static methods:

```java
// Returns className
ClassInfo.docStringId(String className) 
// Returns className + '/' + fieldName
FieldInfo.docStringId(String className, String fieldName) 
// Returns serviceName
ServiceInfo.docStringId(String serviceName) 
// Returns serviceName + '/' + functionName
FunctionInfo.docStringId(String serviceName, String functionName) 
```

and add `docStringId()` member method to them as well. exampleHttpHeaders and exampleRequests could also use `docStringId`. For example:

```
{
  'com.example.FooService': headersA, // common example headers for all functions of FooService
  'com.example.FooService/doBar': headersB, // example headers for FooService.doBar()
  'com.example.FooService': exampleRequest, // ERROR - a service cannot have an example request
  'com.example.FooService/doBar': exampleRequest // OK
} 
```

Sounds good?